engine/tasks.py

```diff
@@ -8,7 +8,8 @@ def run(self, game):
 		raise NotImplementedError("Abstract call.")
 
 
-class ResolutionTask:
+# This has to be a new-style class in order to be able to use super()
```

Comment:
According to http://stackoverflow.com/questions/576169/understanding-python-super-with-init-methods new-style let you write `super().__init__()` instead of `super(ChildB, self).__init__()`, but you're not using this syntax here